src/IssueViz.Security/Taint/TaintStore.cs

```diff
@@ -57,7 +57,7 @@ public void Set(IEnumerable<IAnalysisIssueVisualization> issueVisualizations)
             var oldIssues = taintVulnerabilities;
             taintVulnerabilities = issueVisualizations;
 
-            IssuesChanged?.Invoke(this, new IssuesChangedEventArgs(oldIssues, taintVulnerabilities));
+            IssuesChanged?.Invoke(this, new IssuesChangedEventArgs(oldIssues, taintVulnerabilities.ToList()));
```

Comment:
I hadn't thought about passing an immutable type in the event, but that makes sense too.

It might be better make the store store an immutable collection, then it could be safely passed in the event args.